runner/src/run.rs

```diff
@@ -85,6 +99,11 @@ fn launch_qemu(args: &RunArgs, miralis: PathBuf, firmware: PathBuf) -> ExitCode
     if let Some(cpu) = &cfg.qemu.cpu {
         qemu_cmd.arg("-cpu").arg(cpu);
     }
+
+    if is_qemu_version_6() {
```

Comment:
Let's add a comment above there, to explain why we need to pass an additional flag for some QEMU version.

In general this kind of stuff can not be inferred from the code, so it is super valuable to document why we add those kinds of edge-cases.